src/kakarot/instructions/comparison_operations.cairo

```diff
@@ -418,23 +483,32 @@ namespace ComparisonOperations {
     }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
         alloc_locals;
 
+        let stack_underflow = is_le(ctx.stack.size, 1);
```

Comment:
should we do: 
let popped_items_number = 2;
let stack_underflow = is_le(ctx.stack.size, popped_items_number - 1);
// ...
stack.pop_n(.. n=popped_items_number);?

to avoid errors?